package/src/dep/resolver.rs

```diff
@@ -0,0 +1,69 @@
+use alloc::{collections::BTreeMap, sync::Arc};
+
+use assembly::Library;
+
+use super::Dependency;
+use crate::{Digest, Package};
+
+// DEPENDENCY RESOLUTION
+// ================================================================================================
+
+#[derive(Debug, Clone)]
+pub enum DependencyResolution {
```

Comment:
Would a better name here be `ResolvedDependency`? I tend to read this as a verb, or "the action to resolve".

Similar comment for `LocalResolution`; maybe could be `LocalResolvedDependency` or something

Reply:
Sure. Done in https://github.com/0xPolygonMiden/miden-vm/pull/1544/commits/43e0a83488f491dfbe21305dae257336aa287fd8